### ISSUE TITLE
BarChart: topLabel component not displayed above bars when isAnimated is enabled

### ISSUE THREAD

**Issue by Bordei08** [Abhinandan-Kushwaha/react-native-gifted-charts#813]
### Description

When using the `BarChart` component, I encountered an issue where the `topLabel` component does not display when it is placed above the bars and the `isAnimated` property is set to `true`. However, if the `topLabel` is positioned inside the bars, it displays correctly even with `isAnimated` enabled.

### Steps to Reproduce

1. Use the `BarChart` component in a React Native project.
2. Add data that includes the `topLabel` property positioned above the bars.
3. Set the `isAnimated` property to `true`.
4. Observe that the `topLabel` does not appear above the bars.
5. Move the `topLabel` to be inside the bars and observe that it displays correctly.

```typescript
const barChartData: T_barData[] = [
    {
        stacks: [
            {
                value: 10,
                color: colors.yellow,
                marginBottom: 0,
                gradientColor: colors.yellow,
            },
            {
                value: 15,
                color: colors.darkGreen,
                marginBottom: 0,
                gradientColor: colors.darkGreen,
            }
        ],
        label: 'Test1',
        // In bar label
        topLabelComponent: () => (
            <View style={{ alignItems: 'center' }}>
                <Text
                    style={{
                        color: colors.bogdanRed,
                        fontSize: 12,
                        marginBottom: 1,
                        top: 100,
                        fontFamily: 'PoppinsMedium',
                    }}
                >
                    {Math.round(2)}
                </Text>
                <Text
                    style={{
                        color: colors.white,
                        fontSize: 12,
                        marginBottom: 1,
                        top: 100,
                        fontFamily: 'PoppinsMedium',
                    }}
                >
                    {Math.round(10)}
                </Text>
            </View>
        )
    },
    {
        stacks: [
            {
                value: 12,
                color: colors.yellow,
                marginBottom: 0,
                gradientColor: colors.yellow,
            },
            {
                value: 14,
                color: colors.darkGreen,
                marginBottom: 0,
                gradientColor: colors.darkGreen,
            }
        ],
        label: 'Test2',
        // Above bar label
        topLabelComponent: () => (
            <View
                style={{
                    bottom: 5,
                    alignItems: 'center',
                    flexDirection: 'row',
                    justifyContent: 'space-between',
                    backgroundColor: colors.lightGrey2,
                    padding: 1,
                }}
            >
                <Text
                    style={{
                        color: colors.darkGreen,
                        fontSize: 12,
                        marginBottom: 1,
                        fontFamily: 'PoppinsMedium',
                    }}
                >
                    {Math.round(12)}
                </Text>
                <Text
                    style={{
                        color: colors.bogdanRed,
                        fontSize: 12,
                        marginBottom: 1,
                        fontFamily: 'PoppinsMedium',
                        marginLeft: 4,
                    }}
                >
                    {Math.round(10)}
                </Text>
            </View>
        )
    }
];

return (
    <View>
        <BarChart
            isAnimated
            stackData={barChartData.map((bar) => ({
                ...bar,
                stacks: bar.stacks.map((element: T_value) => ({
                    ...element,
                    value: (element.value < 1 && element.value !== 0 ? element.value + (1 - element.value) : element.value)
                }))
            }))}
            showGradient
            width={chartWidth}
            height={chartHeight}
            frontColor={colors.darkGreen}
            hideYAxisText
            yAxisColor={colors.lightGrey}
            xAxisColor={colors.lightGrey}
            hideRules
            initialSpacing={15}
            showVerticalLines
            spacing={30}
            disableScroll={false}
        />
    </View>
);

```
<img src="https://github.com/user-attachments/assets/d8a05978-c0af-4fbe-9d69-8597c15a349a" alt="description" width="300"/>

### Library Version

- `"react-native-gifted-charts": "^1.4.10"`

**Comment by Abhinandan-Kushwaha** [comment]
@Bordei08 I'm unable to repro it. Can you share the code and data? Also, which version of the library are you using?

**Comment by Bordei08** [comment]
> @Bordei08 I'm unable to repro it. Can you share the code and data? Also, which version of the library are you using?
I have added more details ...

**Comment by Abhinandan-Kushwaha** [comment]
@Bordei08 can you please use the latest version of `react-native-gifted-charts`

**Comment by Bordei08** [comment]
not working...

**Comment by Abhinandan-Kushwaha** [comment]
Hi @Bordei08 This has been fixed in versions `1.4.32` and above. Please use the latest version of the library.
